src/Boilerplates.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace JustCoded\WP\Composer;
+
+use Composer\IO\IOInterface;
+use Composer\Script\Event;
+
+class Boilerplates {
+
+	/**
+	 *  Arguments array
+	 * @var $args
+	 */
+	public static $args = array();
+
+
+	/**
+	 * @param Event $event
+	 *
+	 * @return bool
+	 */
+	public static function theme( Event $event ) {
+		$io = $event->getIO();
+
+		$theme_name = str_replace( 't=', '', $event->getArguments()[1] );
+		$theme_slug = str_replace( ' ', '', $theme_name );
+		$theme_dir  = str_replace( ' ', '', $event->getArguments()[0] );
+		if ( empty( $theme_dir ) ) {
+			$theme_dir = 'wp-content/themes';
+		}
+
+		$src  = 'vendor/justcoded/wordpress-theme-boilerplate';
+		$dst  = $theme_dir . '/' . $theme_slug;
+		self::$args = array(
+			'full_path_to_new_theme' => $dst,
+			'theme_slug'             => $theme_slug,
+			'theme_name'             => $theme_name,
+		);
+
+		if ( is_dir( $src ) ) {
+			$dir = opendir( $src );
+			if ( ! is_dir( $dst ) ) {
+				if ( ! mkdir( $dst, 0777, true ) ) {
+					$io->write( "\n ERROR. Unable to create directory $theme_dir" );
+
+					return false;
+				}
+			}
+
+			self::recurse_copy( $src, $dst );
+		}
+
+	}
+
+
+	/**
+	 * @param: source folder $src
+	 * @param: destination folder $dst
+	 */
+	public static function recurse_copy( $src, $dst ) {
```

Comment:
this method will be used in plugin installation as well.
I suggest to make class inside src/Helpers/FileSystem (the same as init wp command)
and use it

Reply:
done